x-pack/test/alerting_api_integration/security_and_spaces/scenarios.ts

```diff
@@ -94,7 +95,42 @@ const Space1All: User = {
   },
 };
 
-export const Users: User[] = [NoKibanaPrivileges, Superuser, GlobalRead, Space1All];
+const Space1AllWithRestrictedFixture: User = {
+  username: 'space_1_all_with_restricted_fixture',
+  fullName: 'space_1_all_with_restricted_fixture',
+  password: 'space_1_all_with_restricted_fixture-password',
+  role: {
+    name: 'space_1_all_with_restricted_fixture_role',
+    kibana: [
+      {
+        feature: {
+          actions: ['all'],
+          alertsFixture: ['all'],
+          alertsRestrictedFixture: ['all'],
+        },
+        spaces: ['space1'],
+      },
+    ],
+    elasticsearch: {
+      // TODO: Remove once Elasticsearch doesn't require the permission for own keys
+      cluster: ['manage_api_key'],
```

Comment:
There is a `manage_own_api_key` privilege you can grant, which doesn't allow you to grant keys for other users. Would that be more appropriate in the meantime?

Reply:
I have no idea - this isn't new... @mikecote will have context on why this is here I think 😬

Reply:
Yeah, it's a code debt issue for cleanup https://github.com/elastic/kibana/issues/64101 now that we don't need it anymore